From 76f3fac67500e5311e64819b0dc6dd8891f101e5 Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Mon, 15 May 2017 16:23:05 +0300
Subject: [PATCH 1641/2241] fix: crypto: inside-secure: use local dev alias

align the use of dev pointer to other usage in the routine.
this change does not have any functional impact.

Change-Id: I6d2eb7be6a68be8a9ba0380cab36a4d32efc7a2e
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/39485
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/39535
---
 drivers/crypto/inside-secure/safexcel.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/crypto/inside-secure/safexcel.c b/drivers/crypto/inside-secure/safexcel.c
index d111384..5471392 100644
--- a/drivers/crypto/inside-secure/safexcel.c
+++ b/drivers/crypto/inside-secure/safexcel.c
@@ -1244,14 +1244,13 @@ static int safexcel_probe(struct platform_device *pdev)
 			return -EPROBE_DEFER;
 	}
 
-	ret = of_property_read_u32(dev->of_node, "dma-bus-width",
-				   &dma_bus_width);
+	ret = of_property_read_u32(dev->of_node, "dma-bus-width", &dma_bus_width);
 	if (ret) {
 		dev_err(dev, "Failed to read dma-bus-width property\n");
 		goto err_clk;
 	}
 
-	ret = dma_set_mask_and_coherent(&pdev->dev,
+	ret = dma_set_mask_and_coherent(dev,
 					DMA_BIT_MASK(dma_bus_width));
 	if (ret)
 		goto err_clk;
-- 
2.7.4

